README.md

```diff
@@ -39,6 +39,15 @@
     - run: **python main.py all** to run both the generation and simulation.
     - run: **python main.py [simulate|generate] --help** for more help about each option.
     
+    
+## Installing and running under Linux (tested on Debian 10):
+- sudo apt install swig liblzma-dev libbz2-dev
+    - swig - building parasymoblic matrix needs swig
+    - liblzma-dev libbz2-dev - since we use Pandas, we need Python with support for bz2 and lzma support
+        - (https://stackoverflow.com/questions/22346269/bz2-is-module-not-available-when-installing-pandas-with-pip-in-python-virtual) 
+- Building Python 3.8 (currently not on the official repos...)
+    - https://linuxize.com/post/how-to-install-python-3-8-on-debian-10/
```

Comment:
can't we just make a shell script that installs all this?

Reply:
I could!